FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -21,22 +21,22 @@ public final class RemoteFeedLoader: FeedLoader {
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
 		client.get(from: url, completion: { [weak self] result in
 			guard self != nil else { return }
-			completion(FeedLoaderResultMapper.map(result))
+			switch result {
+			case let .success((data, response)):
+				completion(FeedLoaderResultMapper.map(data, response))
+			case .failure:
+				completion(.failure(RemoteFeedLoader.Error.connectivity))
```

Comment:
Since this portion of code is inside the same scope as `RemoteFeedLoader`, we can remove that part from the error to make it shorter:
```suggestion
				completion(.failure(Error.connectivity))
```